tasks/sysctl.yml

```diff
@@ -19,6 +19,10 @@
   command: 'update-initramfs -u'
   when: initramfs.changed
 
+- name: create a combined sysctl-dict if overwrites are defined
+  set_fact: sysctl_config="{{ sysctl_config | combine(sysctl_overwrite) }}"
+  when: sysctl_overwrite | default(omit)
```

Comment:
Have you checked that `omit` evaluates to False? I would not rely on that. Just use `default()` which defaults to `False` :wink: